src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -1322,6 +1564,15 @@ function* iterAnchorsPointsByIndex(anchors, anchorIndices) {
   }
 }
 
+function* iterGuidelinesPointsByIndex(guidelines, guidelineIndices) {
+  if (!guidelineIndices || !guidelines.length) {
+    return;
+  }
+  for (const index of guidelineIndices) {
+    yield guidelines[index];
```

Comment:
There is a bug when one source has more guidelines than others:
- select the last guideline (that doesn't exist in other guidelines)
- then switch to another source

`guidelines[index]` will be undefined in that case, and causes an error when drawing the coordinates. This code should _not_ yield the guideline if it is undefined.

The same bug is in the anchors code, please fix it there, too.